eng/Version.Details.xml

```diff
@@ -154,9 +154,9 @@
       <Sha>53b80940842204f78708a538628288ff5d741a1d</Sha>
     </Dependency>
     <!-- Temporarily pinning Microsoft.Web.Xdt until strict coherency is enabled by default -->
-    <Dependency Name="Microsoft.Web.Xdt" Version="3.1.0" CoherentParentDependency="Microsoft.NET.Sdk" Pinned="true">
-      <Uri>https://github.com/aspnet/xdt</Uri>
-      <Sha>c01a538851a8ab1a1fbeb2e6243f391fff7587b4</Sha>
+    <Dependency Name="Microsoft.Web.Xdt" Version="5.0.0-preview.21431.1" CoherentParentDependency="Microsoft.NET.Sdk" Pinned="true">
```

Comment:
Can this remain at 3.1.0? You might end up with an error when you try to build the stable nuget packages using this dependency (or maybe you don't?), since it will be a non-stable version, but the nuget packages would be stable, which is an error by default. Also note that NuGet only uses `3.0.0`

Reply:
Are you suggesting that we build the updated sha but leave the version at 3.1.0?

Reply:
In the repo yet. I think technically whatever is in this repo doesn't actually matter (this repo doesn't reference the package), but I'm guessing that this feeds into NuGet.Client's dependencies. If this sha produces a 5.0.0-preview, does that imply that NuGet.Client would start depending on the 5.0.0 preview version of this dependency?

Reply:
Also you are the source-build expert so any assumptions I may be making may be wrong :)

Reply:
This is the commit/version sha source-build will produce and therefore it will be the version NuGet.Client picks up during source-build.